src/Features/Core/Portable/PdbSourceDocument/ISourceLinkService.cs

```diff
@@ -21,15 +21,11 @@ internal interface ISourceLinkService
     /// The result of findding a PDB file
     /// </summary>
     /// <param name="PdbFilePath">The path to the PDB file in the debugger cache</param>
-    /// <param name="Status">Status of the operation</param>
-    /// <param name="Log">Any log messages the debugger wrote during the operation</param>
-    /// <param name="IsPortablePdb">Whether the PDB found is portable</param>
-    internal record PdbFilePathResult(string PdbFilePath, string Status, string? Log, bool IsPortablePdb);
+    internal record PdbFilePathResult(string PdbFilePath);
 
     /// <summary>
     /// The result of finding a source file via SourceLink
     /// </summary>
     /// <param name="SourceFilePath">The path to the source file in the debugger cache</param>
-    /// <param name="Log">Any log messages the debugger wrote during the operation</param>
-    internal record SourceFilePathResult(string SourceFilePath, string? Log);
+    internal record SourceFilePathResult(string SourceFilePath);
```

Comment:
Is this just for convenience that these types exist with just a string property? Could we get rid of them?

Reply:
Convenience when these types were split across two branches. My gut says these will be needed in future, as there is still a bunch more features to add to this system, so will leave them for now.